MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,429 @@
+//
+//  MovieDetailsViewController.swift
+//  MovieApp
+//
+//  Created by Bruno Busic on 30.03.2022..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class MovieDetailsViewController: UIViewController {
+    
+    
+    /*
+     Problemi:
+     kad rotiram screen sve mi se unisti
+     u posteru trebam labele dodavati odozdola prema gore
+     kako da dodam ou slicicu favorite
+     kad stavim label.top to bottom of another label ne rade mi inseti i ofseti
+     */
+    
+    
+    
+    private var safeAreaView: UIView!
```

Comment:
Not really sure why you needed this. You could pin each component to `view.safeArea*`.